build/fix-api-docs.js

```diff
@@ -0,0 +1,20 @@
+var replace = require("replace");
+var path = require('path')
+var apiPath = path.join(__dirname, '..', 'docs', 'api');
+
+var replacements = [
+  {find: '\"/docs/guides/\"', replace: '"#"'},
+  {find: '\"/docs/guides/(.*)\.md\"', replace: '"tutorial-$1.html"'},
```

Comment:
should this be the first item in the array? That is, do the most specific changes first and work towards more generic ones after?